docs/data/base/components/button/UnstyledButtonIntroduction/css/index.js

```diff
@@ -54,42 +56,62 @@ function Styles() {
 
   return (
     <style>{`
-  .IntroductionButton {
-    font-family: 'IBM Plex Sans', sans-serif;
-    font-weight: 600;
-    font-size: 0.875rem;
-    line-height: 1.5;
-    background-color: ${cyan[500]};
-    padding: 8px 16px;
-    border-radius: 8px;
-    color: white;
-    transition: all 150ms ease;
-    cursor: pointer;
-    border: 1px solid ${cyan[500]};
-    box-shadow: 0 2px 1px ${
-      isDarkMode ? 'rgba(0, 0, 0, 0.5)' : 'rgba(45, 45, 60, 0.2)'
+      .Button {
+        all: unset;
+        align-items: center;
+        background-clip: padding-box;
+        background-color: ${cyan[500]};
+        border-color: ${cyan[500]};
+        border-radius: .5rem;
+        border-style: solid;
+        border-width: .063rem;
+        box-shadow: 0 1px 2px ${
+          isDarkMode ? 'rgba(0, 0, 0, 0.15)' : 'rgba(60, 86, 118, 0.2)'
+        }, 0 2px 3px ${
+      isDarkMode ? 'rgba(0, 0, 0, 0.15)' : 'rgba(60, 86, 118, 0.2)'
     }, inset 0 1.5px 1px ${cyan[400]}, inset 0 -2px 1px ${cyan[600]};
-  }
-  .IntroductionButton:hover {
-    background-color: ${cyan[600]};
-  }
-  .IntroductionButton.base--active {
-    background-color: ${cyan[700]};
-    box-shadow: none;
-    transform: scale(0.99);
-  }
-  .IntroductionButton.base--focusVisible {
-    box-shadow: 0 0 0 4px ${isDarkMode ? cyan[300] : cyan[200]};
-    outline: none;
-  }
-  .IntroductionButton.base--disabled {
-    background-color: ${isDarkMode ? grey[700] : grey[200]};
-    color: ${isDarkMode ? grey[200] : grey[700]};
-    border: 0;
-    cursor: default;
-    box-shadow: none;
-    transform: scale(1);
-  }
+        box-sizing: border-box;
+        color: white;
+        display: inline-flex;
+        flex-shrink: 0;
+        font-family: 'IBM Plex Sans', sans-serif;
+        font-size: .875rem;
+        font-weight: 600;
+        height: 2.5rem;
```

Comment:
I usually try to avoid setting fixed height values for buttons, even though nothing seems to break after zooming in on the page. I've stopped doing that after I read [this blog post](https://ashleemboyer.com/blog/don-t-use-fixed-css-height-or-width-on-text-containers) — what do you think?

Reply:
The height value is using `rem` so it's fluid. When you zoom in, or increase your default browser text size, all values will update accordingly.

I always recommend defining a `height` so that all UI controls can be sized consistently, and adhere to a baseline grid. And also so that they don't inadvertently change size when you add in an icon for example.